port_ocean/clients/port/mixins/integrations.py

```diff
@@ -99,15 +100,24 @@ async def patch_integration(
         handle_status_code(response)
         return response.json()["integration"]
 
+    def sirealize_logs(self,logs: list[dict[str, Any]]):
+        _logs = copy.deepcopy(logs)
+        for log in _logs:
+            if 'extra' in log.keys() and 'exc_info' in log['extra'].keys() and isinstance(log['extra']['exc_info'],Exception):
+                sirealized_exception = ''.join(format_exception(log['extra']['exc_info']))
```

Comment:
Does it also prints it in the same way? with the appropriate `\n` and all the spaces? 
```suggestion
                serialized_exception = ''.join(format_exception(log['extra']['exc_info']))
```